youthful-piety/modernized/06-chapter-05.adoc

```diff
@@ -372,6 +377,8 @@ and her resignation to his blessed will--and also gave good
 advice to her relations and others who came too see her.
 A few of those noted down by her mother, are as follows:
 
+[quote]
```

Comment:
Not sure if this should be {quote] or not? And after this, there are several more places where the there's a date with words and things recorded about her, but I didn't think all those should be in [quote]'s?

Reply:
I don't think these need to be marked as [quotes]... in part because they aren't really quotations from a book, paper, testimony, etc., and also because (like you said) there are a lot more of her words scattered throughout this section that aren't marked as a [quote].

Reply:
Haha, see what I mean about wanting to embed when I shouldn't??!! 😆